src/CommunityToolkit.Maui.Core/Primitives/Defaults.shared.cs

```diff
@@ -5,6 +5,7 @@ namespace CommunityToolkit.Maui.Core;
 /// <summary>
 /// Default Values for CommunityToolkit.Maui
 /// </summary>
+[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
I assumed that we preferred to keep it `internal` since it's only used internally for default values in the toolkit. But since it lives in `CommunityToolkit.Maui.Core` and we needed to leverage it in `CommunityToolkit.Maui`, we had to make it `pubic`.

That's just a guess though.